monitoring/src/env.ts

```diff
@@ -5,3 +5,8 @@ export const envStage: string = process.env['STAGE']
 export const envJurisdiction: string | undefined =
 	process.env['CMP_JURISDICTION'];
 export const envAwsRegion: string | undefined = process.env['AWS_REGION'];
+
+// This is the DCR endpoint
+export const localBaseURL: string = process.env['LOCAL_BASE_URL']
+	? process.env['LOCAL_BASE_URL']
+	: 'http://localhost:3030';
```

Comment:
I think it's worth being extra clear in the README that this only works with DCR and not frontend.